iogt_content_migration/management/commands/load_v1_db.py

```diff
@@ -678,6 +692,7 @@ def create_footer(self, row):
             last_published_at=row['last_published_at'],
             search_description=row['search_description'],
             seo_title=row['seo_title'],
+            commenting_status=CommentStatus.CLOSED
```

Comment:
V1 footers also have `commenting_starts_at` and `commenting_ends_at`.

Reply:
Yes, but they are nullable, so we'll just leave them as is. right?

Reply:
But this data is actually populated in V1, so we should port it over.